omnibus/package-scripts/agent/postinst

```diff
@@ -91,23 +91,37 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
             semodule -v -i $CONFIG_DIR/selinux/system_probe_policy.pp >/dev/null 2>&1
             if [ "$?" != "0" ]; then
-                echo "The SELinux policy module for datadog-agent is not compatible with this host."
-                echo "Please install or update the selinux-policy-default package and then reinstall the datadog-agent package to configure SELinux for the Datadog Agent."
-                echo "Skipping SELinux configuration."
+                echo "Couldn’t load system-probe policy."
+                echo "To be able to run system-probe on your host, please install or update the"
+                echo "selinux-policy-default and policycoreutils-python-utils packages."
+                echo "Then run the following commands:"
+                echo "    semodule -i $CONFIG_DIR/selinux/system_probe_policy.pp"
+                echo "    semanage fcontext -a -t system_probe_t $INSTALL_DIR/embedded/bin/system-probe"
+                echo "    restorecon -v $INSTALL_DIR/embedded/bin/system-probe"
             else
                 echo "Labeling SELinux type for the system-probe binary."
                 if command -v semanage >/dev/null 2>&1 && command -v restorecon >/dev/null 2>&1;then
                     semanage fcontext -a -t system_probe_t $INSTALL_DIR/embedded/bin/system-probe && restorecon -v $INSTALL_DIR/embedded/bin/system-probe
                     if [ "$?" != "0" ]; then
-                        echo "Could not relabel the system-probe binary with type system_probe_t."
+                        echo "Couldn’t install system-probe policy."
+                        echo "To be able to run system-probe on your host, please install or update the"
+                        echo "selinux-policy-default and policycoreutils-python-utils packages."
+                        echo "Then run the following commands:"
+                        echo "    semodule -i $CONFIG_DIR/selinux/system_probe_policy.pp"
+                        echo "    semanage fcontext -a -t system_probe_t $INSTALL_DIR/embedded/bin/system-probe"
+                        echo "    restorecon -v $INSTALL_DIR/embedded/bin/system-probe"
                     fi
                 else
-                    echo "Could not set up SELinux labels for datadog-agent: semanage and/or restorecon commands not found."
-                    echo "Please make sure they are installed (via the policycoreutils, policycoreutils-python and/or policycoreutils-python-utils packages), and then run:"
+                    echo "Couldn’t load system-probe policy (missing selinux utilities)"
+                    echo "To be able to run system-probe on your host, please install or update the"
+                    echo "selinux-policy-default and policycoreutils-python-utils packages."
+                    echo "Then run the following commands:"
+                    echo "    semodule -i $CONFIG_DIR/selinux/system_probe_policy.pp"
```

Comment:
I guess this line is not needed? It wasn't there before. Unless you want to display the same instructions for all error cases.

Reply:
In the future (once this is refactored), all cases will show the same lines, so I think we can start doing this right now. Loading the module twice is not a problem (modules are named and versioned, so the loading the same module twice is idempotent).